languagetool-language-modules/ar/src/main/resources/org/languagetool/resource/ar/confusion_sets.txt

```diff
@@ -0,0 +1,14 @@
+# Arabic confusion sets
+#
+# Line format:
+# <word1>|<description1>; <word2>|<description2>; <factor>   # optional comment
+#   <word1> and <word2> are words that can easily be confused
+#   <description> will be used in the error message to explain the word (optional)
+#   <factor> is the factor of how much more the other word must be more
+#            probable so the text is considered potentially incorrect.
+#            Use a higher value for better precision but lower recall.
+# Order is relevant for ambiguous cases like 'know' ('no' or 'now') where the match
+# is used whose pair comes first in this file.
+
+ضن; ظن; 1000
```

Comment:
Do you have ngrams for Arabic? Or does this work without ngrams?

Reply:
Not for now :(